tests/admin_custom/apps.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import unicode_literals
```

Comment:
I think adding another test folder is slightly frowned upon, especially for just one test. This could maybe fit under `admin_registration`?